packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -177,73 +177,74 @@ const AppModals = observer(() => {
             }
             break;
     }
+    if (!url_action_param) {
+        if (
+            is_logged_in &&
```

Comment:
Hi @amina-deriv 
Just for more readability can we store this condition in variable and we use that variable here for checking condition 😬